tests/UI/pages/BO/catalog/productsV2/index.ts

```diff
@@ -374,43 +383,82 @@ class Products extends BOBasePage {
   }
 
   /**
-   * Click on bulk delete products
+   * Click on bulk action products
    * @param page {Page} Browser tab
+   * @param action {string} Enable/disable/duplicate or delete
    * @returns {Promise<string>}
    */
-  async clickOnBulkDeleteProducts(page: Page): Promise<string> {
+  async clickOnBulkActionsProducts(page: Page, action: string): Promise<string> {
+    let textResult;
     await Promise.all([
       await this.waitForSelectorAndClick(page, this.productBulkMenuButton),
       await this.waitForVisibleSelector(page, this.bulkActionsDropDownMenu),
     ]);
-    await this.waitForSelectorAndClick(page, this.bulkActionsDeleteSelectionLink);
-    await this.waitForVisibleSelector(page, this.modalBulkDeleteProducts);
 
-    return this.getTextContent(page, this.modalBulkDeleteProductsBody);
+    if (action === 'enable' || action === 'disable') {
+      await this.waitForSelectorAndClick(page, this.bulkActionsSelectionLink(`${action}_selection`));
```

Comment:
same : 
I think maybe you can have a function that return the value to put on the selector and use it on each function

Reply:
Done thx :+1: